node/src/main/resources/application.conf

```diff
@@ -346,9 +346,7 @@ waves {
     supported = []
   }
 
-  extensions = [
-    # com.wavesplatform.matcher.Matcher
-  ]
```

Comment:
Listing all our plugins here seemed like a good idea. Is there a reason to change this?